src/patternfly/components/ChipGroup/examples/chip-group-toolbar-expanded-example.hbs

```diff
@@ -6,28 +6,32 @@
     {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id '-chip_one"')}}
       Chip One
     {{/chip-text}}
-    {{#> button button--modifier="pf-m-plain" button--HasTimesCircleIcon="true" button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_one ' chip-group--id 'chip_one" aria-label="Remove" id="' chip-group--id 'remove_chip_one"')}}
+    {{#> button button--modifier="pf-m-plain"  button--attribute=(concat 'aria-labelledby="' chip-group--id 'remove_chip_one ' chip-group--id 'chip_one" aria-label="Remove" id="' chip-group--id 'remove_chip_one"')}}
```

Comment:
can you remove the extra space before `button--attribute`?

Reply:
spacing adjusted.